### DISCUSSION TITLE
Add analytics for MySQL prepared statements

### DISCUSSION THREAD

**Discussion by drupaladmin** [Releem/releem-agent#370] [Feature Requests]
Add analytics for MySQL prepared statements

Related issue #353

**Comment by kochetovd** [comment]
The problem of calculating Latency using prepared statements and their showing in query analytics is the approach of using prepared queries - when processing requests, the application first creates a prepared stmt, executes it and then destroys it. Therefore, it is rather difficult to get statistics of all executed queries. 
In addition, for prepared statements it is not specified to which database the query was executed and how unique it is. 
Also there were problems that for prepared statements in the table Performance_schema not recorded statistics of query execution, in the table events_statements_* get EXECUTE queries executed in the console mysql, the execution of prepared statements, but then there are difficulties converting them into normal sql query. When executing prepared queries through mysql binary protocols, information about their execution is not saved anywhere.

**Comment by drupaladmin** [comment]
We've published discussion to MariaDB developers: https://lists.mariadb.org/hyperkitty/list/developers@lists.mariadb.org/thread/3OP65GIDEOKZHQYN55ZGCNIPZDKOXU3V/
